src/synapse/admin.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python3
+
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+"""Helper module used to manage admin tasks involving Synapse API and Workload."""
+
+import logging
+import typing
+from secrets import token_hex
+
+import ops
+
+from user import User
+
+from .api import register_user
+from .workload import get_registration_shared_secret
+
+logger = logging.getLogger(__name__)
+
+
+def create_admin_user(container: ops.Container) -> typing.Optional[User]:
```

Comment:
Nice to have this outside of the charm.py file!

Reply:
Thanks!